src/ShopBundle/objects/TShopBasket/TShopBasketCore.class.php

```diff
@@ -861,7 +861,7 @@ protected function RecalculateVAT()
         $this->GetActiveVATList()->GoToStart();
 
         $oShopConf = TdbShop::GetInstance();
-        if ($oShopConf->fieldShippingVatDependsOnBasketContents) {
+        if (true === $oShopConf->fieldShippingVatDependsOnBasketContents && $this->dCostVATWithoutShipping > 0.001) {
```

Comment:
Was ist 0.001 - ggfs. benannte Konstante verwenen?

Reply:
Das ist einfach ein größer 0 Aber Stefan meinte mal das sollte man 0.001 verwenden siehe auch andere Abfragen in der Klasse.

Reply:
<= 0.001 habe ich gefunden.

Da macht das mathematisch/programmiersprachlich vermutlich Sinn.

Hier dürfte aber > 0 richtig(er) sein.

Reply:
Changed